### DISCUSSION TITLE
Unable to build controller docker

### DISCUSSION THREAD

**Discussion by HemanthNRao** [linkerd/linkerd2#11045] [Help]
I am trying to build controller docker with the docker file available at https://github.com/linkerd/linkerd2/blob/main/controller/Dockerfile the build was successful but when I tried to create a container out of it I am getting the below error
**expected a subcommand**

**Comment by adleong** [comment]
Hi @HemanthNRao 

Can you tell us a bit more about what you're trying to do?  The Linkerd controller docker images are intended to be used with the Linkerd Helm charts.
